installation/proc-install-rhdh-operator-admin.adoc

```diff
@@ -40,14 +40,29 @@ curl -L https://gist.githubusercontent.com/nickboldt/d54ba9d7264a5480d14eea6c3bd
 ----
 
 .Procedure
-. In the *Administrator* perspective of the OpenShift web console, go to *Operators* → *OperatorHub*, search for Red Hat Developer Hub, and install the Red Hat Developer Hub Operator. For more info, see link:https://docs.openshift.com/container-platform/4.14/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-from-operatorhub-using-web-console_olm-adding-operators-to-a-cluster[Installing from OperatorHub using the web console].
+. In the *Administrator* perspective of the OpenShift web console, go to *Operators* → *OperatorHub*, search for Red Hat Developer Hub, and install the Red Hat Developer Hub Operator. For more info, see link:https://docs.openshift.com/container-platform/4.14/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-from-operatorhub-using-web-console_olm-adding-operators-to-a-cluster[Installing from OperatorHub using the web console]. It is recommended to deploy the Backstage Operator in a dedicated default namespace `rhdh-operator`. The cluster administrator can restrict other users the access to the operator resources through RoleBindings or ClusterRoleBindings. You can choose to deploy the operator in the `openshift-operators` namespace instead. However, you should keep in mind that the Backstage Operator shares the namespace with other operators and therefore any users who can create workloads in that namespace can get their privileges escalated from all operators' service accounts.
```

Comment:
if you create the `operator` in `openshift-operators` (per the recommended guidelines for an `AllNamespaces` subscription) there's no need to create the `backstage` CRs in the same namespace. 

```
The namespace to which you subscribe the Operator must have an Operator group that matches the install mode of the Operator, either the AllNamespaces or SingleNamespace mode. If the Operator you intend to install uses the AllNamespaces mode, then the openshift-operators namespace already has an appropriate Operator group in place.

However, if the Operator uses the SingleNamespace mode and you do not already have an appropriate Operator group in place, you must create one.
```

https://docs.openshift.com/container-platform/4.14/operators/user/olm-installing-operators-in-namespace.html

Reply:
@nickboldt did you mean backstage Operator Group instead? The downside of deploying the operator in openshift-operators is it shares the namespace with other operators. So if a user ever gets the permission to create workloads in that namespace, he would be be able to get his privileges escalated from all the service accounts in that namespace. Deploying the operator in a separate namespace minimizes such privilege escalation.